graylog2-web-interface/src/views/components/aggregationbuilder/AggregationControls.test.jsx

```diff
@@ -3,7 +3,6 @@ import * as React from 'react';
 import * as Immutable from 'immutable';
 import { mount } from 'wrappedEnzyme';
 import { render } from 'wrappedTestingLibrary';
-import '@testing-library/jest-dom/extend-expect';
```

Comment:
Don't we need this for [`toHaveTextContent`](https://github.com/testing-library/jest-dom#tohavetextcontent) on line 38?

Reply:
Oh wait, that's right, it was moved to wrappedTestingLibrary.